editor/components/post-publish-panel/postpublish.js

```diff
@@ -48,26 +49,35 @@ class PostPublishPanelPostpublish extends Component {
 	}
 
 	render() {
-		const { post, postType } = this.props;
+		const { isScheduled, post, postType } = this.props;
 		const viewPostLabel = get( postType, [ 'data', 'labels', 'view_item' ] );
 
+		const postPublishNonLinkHeader = isScheduled ?
+			<Fragment>{ __( 'is now scheduled. It will go live on' ) } <PostScheduleLabel />.</Fragment> :
+			__( 'is now live.' );
+		const postPublishBodyText = isScheduled ?
+			__( 'The post address will be:' ) :
+			__( 'What\'s next?' );
+
 		return (
 			<div className="post-publish-panel__postpublish">
 				<PanelBody className="post-publish-panel__postpublish-header">
-					<a href={ post.link }>{ post.title || __( '(no title)' ) }</a>{ __( ' is now live.' ) }
+					<a href={ post.link }>{ post.title || __( '(no title)' ) }</a> { postPublishNonLinkHeader }
```

Comment:
We should not concatenate translated strings, because the order of the concatenated parts can change in localization:

>Use format strings instead of string concatenation—sprintf(__('Replace %1$s with %2$s'), $a, $b); is always better than __('Replace ').$a.__(' with ').$b; .

https://codex.wordpress.org/I18n_for_WordPress_Developers#Best_Practices

Reply:
To preempt: It is unsolved this need to include React elements in translated strings. We can look to some other efforts as prior art:

https://github.com/Automattic/interpolate-components

But if we don't want to solve the larger problem, we're probably better off not localizing at all here.

Reply:
We should definitely fix this — cc @jorgefilipecosta

Reply:
I might take it upon myself to look at what would be needed for component interpolation.

Reply:
Description of the issue / task at #9846